base/collection.go

```diff
@@ -546,14 +547,14 @@ func (c *Collection) Dump() {
 
 func (c *Collection) GetStatsVbSeqno(maxVbno uint16, useAbsHighSeqNo bool) (uuids map[uint16]uint64, highSeqnos map[uint16]uint64, seqErr error) {
 
-	agent, agentErr := c.Bucket().Internal().IORouter()
+	agent, agentErr := c.getGoCBAgent()
 	if agentErr != nil {
 		return nil, nil, agentErr
 	}
 
 	statsOptions := gocbcore.StatsOptions{
 		Key:      "vbucket-seqno",
-		Deadline: time.Now().Add(5 * time.Minute),
+		Deadline: getDCPGoCBDeadline(),
```

Comment:
Should this be using a deadline that's different than getBucketOpDeadline?  It seems functionally equivalent, and might be preferable to have it hooked into the existing configurability.

Reply:
Talked this over with @bbrks and the idea is that if something can't start up a DCP stream, we should give it a longer timeout but use the user configurable timeout for document gets.

Reply:
Decided to use standard bucket op timeout.